pkg/rpcclient/unwrap/unwrap.go

```diff
@@ -208,6 +220,9 @@ func ArrayAndSessionIterator(r *result.Invoke, err error) ([]stackitem.Item, uui
 	itm := r.Stack[0]
 	arr, ok := itm.Value().([]stackitem.Item)
 	if !ok {
+		if itm.Type() == stackitem.AnyT {
+			return nil, uuid.UUID{}, result.Iterator{}, ErrNull
+		}
```

Comment:
This `itm` encapsulates the result of `CreateCallAndPrefetchIteratorScript`, in particular, the result of `emitCallAndUnwrapIteratorScript`. In case if contract returns an improper value, this script will FAULT VM execution. Hence, I'd say that this check here is useless.

Reply:
This function can be applied to any `*result.Invoke`, even erroneously, so it doesn't hurt either, at least this makes the behavior consistent with other functions.